firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -83,26 +83,37 @@ void home()
 // Moves the magnet from the "start" point to the "end" point
 // This can only move in straight lines
 // Returns a boolean indicating success/error
-bool moveStraight(int motor[], int startCol, int startRow, int endCol, int endRow)
+bool moveStraight(int motor[], float startCol, float startRow, float endCol, float endRow)
 {
   // A specific motor is passed to this function since we are only moving one here
 
   // How many steps per space
-  int spaces, dir, numSteps;
+  float pieceSpaces;
+  int dir, numSteps;
   int i;
 
+  // Makes sure all arugments produce a multiple of unitspaces
```

Comment:
Typo: should be `argument`

Reply:
I'm leaving this one as is because it's part of Aden's changes that were rebased in. I don't want to cause additional merge conflicts by renaming it in my changes.